### PR TITLE
Minor project settings

### PR DESCRIPTION
**Pull request by iccaprar**
Environment setup variables

### CODE REVIEW

**Comment by georgebejan** [comment]
@iccaprar  
What's the purpuse for those scripts?
As far as I can tell, they'll work only on linux environments

**Comment by iccaprar** [comment]
Linux and MacOS, yes. Just helps in setting up the environment instead of modifying shell profiles.

**Comment by mihairinzis** [comment]
i don't think we really need those os-specific scripts because those vars will be set in the dockerfile

File: api/build.gradle [hunk 1]
```diff
@@ -8,6 +8,7 @@ plugins {
 group = 'com.code4ro'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
+targetCompatibility = '11'
```
Comment (chalx) [inline api/build.gradle:11]: Why not 13?

Comment (iccaprar) [reply api/build.gradle:11]: source = 11

**Comment by iccaprar** [comment]
Removed the scripts

File: api/build.gradle (hunk 1 above)
Comment (chalx) [reply api/build.gradle:11]: Yes, but we can use Java 13.
sourceCompatibility=13
targetCompatibility=13

Comment (iccaprar) [reply api/build.gradle:11]: I can do the change if others agree

Comment (RaduCStefanescu) [reply api/build.gradle:11]: I think we can live with 11

Comment (georgebejan) [reply api/build.gradle:11]: We choose 11 because it's the latest LTS

**Comment by iccaprar** [comment]
Closed for a cleaner one.

**Pull request closed without merging**